test/book/specialized/holt-ren-tropical-geometry/variety.jlcon

```diff
@@ -8,7 +8,7 @@ julia> I = intersect(ideal([x+y+z+1,2*x+11*y+23*z+31]),
                      ideal([t^3*x*y*z-1]));
 
 julia> TropV = tropical_variety(I,nu)
-2-element Vector{TropicalVariety}:
+2-element Vector{TropicalVariety{typeof(min),true}}:
```

Comment:
I think the problem is a missing whitespace:
```suggestion
2-element Vector{TropicalVariety{typeof(min), true}}:
```
See here: https://github.com/oscar-system/Oscar.jl/actions/runs/14125235041/job/39572795213?pr=4061#step:7:476

Reply:
Thanks, that's fixed now!